ruby/rollbar.rb

```diff
@@ -0,0 +1,10 @@
+Rollbar.configure do |config|
+  config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
```

Comment:
Als je aan het ontwikkelen bent zou je het misschien nog niet willen gebruiken, maar dat is meer een kwestie van je Rails environment goed configureren. Ik ga er vanuit dat als je deze generator uitvoert, je Rollbar wil gebruiken. En ik denk dat een productie-app zonder exception monitoring niet acceptabel is. Of in ieder geval: een productie-app mét de Rollbar-gem én wat configuratie, maar waarbij het vervolgens stiekem _niet_ werkt.